packages/sandbox/src/components/FieldPluginSandbox.tsx

```diff
@@ -252,6 +267,17 @@ const useSandbox = (
       }),
     [uid, dispatchContextRequest, story],
   )
+
+  const onUserContextRequested = useCallback(
+    () =>
+      dispatchUserContextRequest({
+        uid,
+        action: 'get-user-context',
+        user,
+      }),
+    [uid, dispatchUserContextRequest, user],
+  )
```

Comment:
Hey @Dawntraoz 🙌 

I guess here we would need to pass the received `callbackId` back to the field plugin so it can resolve the call and remove it from the `callbackQueue`

```suggestion
  const onUserContextRequested = useCallback(
    (message: GetUserContextMessage) =>
      dispatchUserContextRequest({
        uid,
        action: 'get-user-context',
        user,
        callbackId: message.callbackId,
      }),
    [uid, dispatchUserContextRequest, user],
  )
```